header1.h

```diff
@@ -0,0 +1,90 @@
+
+using namespace std;
+
+enum governmentType
+{
+    DEMOCRACY,
+    REPUBLIC,
+    AUTOCRACY,
+    OLIGARCHY,
+    ARISTOCRACY,
+    MONARCHY
+};
+
+
+class Country
+{
+private:
+    string name;
+    string capital;
+    float area;
+    int code;
+    governmentType Type;
+
+public:
+    int gdp;
+    int population;
+
+    Country()
+    {
+    }
+
+    Country(string name, string capital, int code, int population, float area, int gdp)
+    {
+        this->name = name;
+        this->capital = capital;
+        this->code = code;
+        this->population = population;
+        this->area = area;
+        this->gdp = gdp;
+    }
+
```

Comment:
get and set functions should be named  like variables. Exemple string get_name();